Do we want to do something with the failing CI code static analysis check?
```
prefix=components/odh-notebook-controller --timeout=5m] in [/home/runner/work/kubeflow/kubeflow/components/odh-notebook-controller] ...
  Error: components/odh-notebook-controller/controllers/notebook_route.go:106:21: ST1023: should omit type bool from declaration; it will be inferred from the right-hand side (staticcheck)
  	var isGenerateName bool = false
  	                   ^
  Error: components/odh-notebook-controller/controllers/notebook_route.go:236:5: QF1008: could remove embedded field "ObjectMeta" from selector (staticcheck)
  	sa.ObjectMeta.Annotations["serviceaccounts.openshift.io/oauth-redirectreference.second"] = "" +
  	   ^
  2 issues:
  * staticcheck: 2
  ```
AFAIK it's not failing in main?

--- a/components/notebook-controller/controllers/notebook_controller.go
+++ b/components/notebook-controller/controllers/notebook_controller.go
@@ -144,7 +144,7 @@ func (r *NotebookReconciler) Reconcile(ctx context.Context, req ctrl.Request) (c
 	// controller-service-hash that shouldn't exceed 63 chars.
 	isGenerateName := false
 	if len(instance.Name) > MaxStatefulsetNameLength {
-		log.Info("Notebook name is too long, it should be less than 52 chars")
+		log.Info("Notebook name is too long, it can be {MaxStatefulsetNameLength} chars long at most")
 		isGenerateName = true
 	}
 

--- a/components/odh-notebook-controller/controllers/notebook_route.go
+++ b/components/odh-notebook-controller/controllers/notebook_route.go
@@ -103,7 +103,7 @@ func (r *OpenshiftNotebookReconciler) reconcileRoute(notebook *nbv1.Notebook,
 	// Initialize logger format
 	log := r.Log.WithValues("notebook", notebook.Name, "namespace", notebook.Namespace)
 
-	var isGenerateName bool = false
+	var isGenerateName = false
 	// If the route name + namespace is greater than 63 characters, the route name would be created by generateName
 	// ex: notebook-name 48 + namespace(rhods-notebooks) 15 = 63
 	if len(notebook.Name)+len(notebook.Namespace) > RouteSubDomainMaxLen {
@@ -233,7 +233,7 @@ func (r *OpenshiftNotebookReconciler) ReconcileRoute(
 
 // InsertSecondRedirectReference inserts the second redirect reference into the ServiceAccount
 func InsertSecondRedirectReference(sa *corev1.ServiceAccount, routeName string) *corev1.ServiceAccount {
-	sa.ObjectMeta.Annotations["serviceaccounts.openshift.io/oauth-redirectreference.second"] = "" +
+	sa.Annotations["serviceaccounts.openshift.io/oauth-redirectreference.second"] = "" +
 		`{"kind":"OAuthRedirectReference","apiVersion":"v1",` +
 		`"reference":{"kind":"Route","name":"` + routeName + `"}}`
 	return sa

